examples/distro/instrumentation/build.gradle

```diff
@@ -54,6 +54,6 @@ shadowJar {
   relocate "io.opentelemetry.context", "io.opentelemetry.javaagent.shaded.io.opentelemetry.context"
 
   // relocate OpenTelemetry extensions
+  relocate "io.opentelemetry.extension.aws", "io.opentelemetry.javaagent.shaded.io.opentelemetry.extension.aws"
   relocate "io.opentelemetry.extension.kotlin", "io.opentelemetry.javaagent.shaded.io.opentelemetry.extension.kotlin"
-  relocate "io.opentelemetry.extension.trace.propagation", "io.opentelemetry.javaagent.shaded.io.opentelemetry.extension.trace.propagation"
```

Comment:
do we still need this for b3, jaeger, and ottrace?

Reply:
We don't directly use any in our instrumentation so I think it's ok not to shade them (they're only in the agent classloader, never injected into the user). Maybe it's confusing though let me know if we should just shade for the heck of it.

Reply:
omg it's so confusing 😭, but let's not shade for the heck of it, that just masks the confusion of what's going on, I'll try to write a better comment for these relocation blocks tomorrow, let's merge this(!)